src/pages/RFOX/components/Stake/StakeConfirm.tsx

```diff
@@ -152,7 +152,10 @@ export const StakeConfirm: React.FC<StakeConfirmProps & StakeRouteProps> = ({
     isSuccess: isNewContractBalanceOfCryptoBaseUnitSuccess,
   } = useStakingBalanceOfQuery<string>({
     stakingAssetId: confirmedQuote.stakingAssetId,
-    stakingAssetAccountAddress: getStakingContract(confirmedQuote.stakingAssetId),
+    accountId: toAccountId({
+      account: getStakingContract(confirmedQuote.stakingAssetId),
+      chainId: fromAssetId(confirmedQuote.stakingAssetId).chainId,
+    }),
```

Comment:
Yes, this is not an `AccountId` that's in the store, but as far as CAIP things go, that's still a very valid `AccountId`!